eks_cluster.tf

```
@@ -1,5 +1,5 @@
 ###########################################
-###...Creation Kubernetes Cluster
+### Kubernetes Cluster. Step 2          ###
 ###########################################
 
 module "kubcluster" {
```

```suggestion
module "kubecluster" {
```
shortcut for Kubernete sis kube, not kub

---

Fixed in next PR.